website/data/docs-nav-data.json

```diff
@@ -2004,10 +2004,6 @@
             "title": "Lightrun",
             "href": "/plugins/drivers/community/lightrun"
           },
-          {
-            "title": "LXC",
-            "href": "/plugins/drivers/community/lxc"
-          },
```

Comment:
Should we remove the `rkt` link while we're here? That's are dead and just redirects back to the plugins / community page.

Reply:
Removed this and other rkt mentions within https://github.com/hashicorp/nomad/pull/24107/commits/fa5ab74bdaacf5a68bd4537866afd151651f66e2